clang/test/SemaSYCL/remove-restriction-builtin-assume-aligned.cpp

```diff
@@ -0,0 +1,21 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -sycl-std=2020 -verify -pedantic -fsyntax-only %s
```

Comment:
Is `-pedantic `required in this test?

Reply:
It isn't 'required' for this test per se, but I have used it before to make sure clang issues all warnings. Used it as an additional layer of checking if the test would emit any diagnostics, but it is not needed in hindsight.